release_guide/12/1_Staging_preparation.adoc

```diff
@@ -127,8 +127,8 @@ https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Devstudio/view
 
 Currently:
 
-version_jbt_IS=4.6.0.Final
-version_ds_IS=12.0.0.GA
+ version_jbt_IS=4.6.0.Final
```

Comment:
Latest version is 4.11.0.Final see https://download.jboss.org/jbosstools/photon/stable/updates/integration-stack/
For Studio 12.11.0.Final see https://devstudio.redhat.com/12/stable/updates/integration-stack

Reply:
fixed.